lighthouse-core/audits/manifest-display.js

```diff
@@ -57,7 +57,7 @@ class ManifestDisplay extends Audit {
       rawValue: hasRecommendedValue,
       displayValue
     };
-    if (!hasRecommendedValue) {
+    if (manifest && !hasRecommendedValue) {
```

Comment:
This new check will only show the "Manifest display property should be set" debug string if there is a manifest and the display property is missing.  Here's what it looks like when a manifest is missing:

![screen shot 2017-02-01 at 12 17 31 pm](https://cloud.githubusercontent.com/assets/238208/22524557/8a7c9470-e878-11e6-9611-cd42b1a3d81b.png)